### DISCUSSION TITLE
IS there a way to disable goguardian block but keep the extension working so I can use it for rigtools since it the only one with the right  permissions , bookmark let disabled .

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1385] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
???

**Comment by akabutnicer** [comment]
did I not just answer this question once before? if not, **no**, you **cannot**.

**Reply by ghost** [reply]
when

**Reply by ghost** [reply]
then how could swamp do it then

**Reply by ghost** [reply]
your supposed to be nice too , its part of your user name

**Reply by akabutnicer** [reply]
I didn't come off mean but ok. anyways I might have misinterpreted your message, are you saying you can use swamp, but you want a way you can 'disable' goguardian without actually turning off the extension? if so, in the uh evaluate code thing or whatever, enter `chrome.tabs.updateAsync = null` in the background field

**Reply by ghost** [reply]
No, I was wondering how swamp could soft disable goguardian. what does that do , do you want me to run it in a another's extension evualted code area? Thanks

**Reply by ghost** [reply]
What I mean by what does that is that code what does it do?

**Reply by akabutnicer** [reply]
> No, I was wondering how swamp could soft disable goguardian. what does that do , do you want me to run it in a another's extension evualted code area? Thanks

that's...what I said. also when you load up swamp, is there like an 'evaluate code' section perchance?

**Reply by ghost** [reply]
I don’t have access to swamp , but yes there is a evalted code area .

**Reply by akabutnicer** [reply]
yeah fair enough. forget swamp just run `chrome.extension.getBackgroundPage().chrome.tabs.updateAsync = null` via the 'evalted code area'...

**Reply by schoolexploitkid** [reply]
swamp: chrome.extension.getBackgroundPage().close()

**Comment by ghost** [comment]
Where do I RUn that?

**Reply by AshtonDavies** [reply]
evaluated code area... as already established well before you posting this comment

**Comment by ghost** [comment]
dosent work

**Comment by 88411** [comment]
Look at my most recent post I made. It is called interesting go guardian glitch with chrome://restart

**Comment by 88411** [comment]
![image](https://github.com/user-attachments/assets/b2a65429-2df4-4535-8423-ec598c8ca0b0)


Try clicking forms locked mode then click disable all extensions switch

**Reply by 88411** [reply]
Nvm I'm stupid. You can go to swamp they go guardian then enable dns emulator then it disables

**Reply by Noobster5** [reply]
Hello, how did you get there? I'm running ChromeOS v130

**Reply by 88411** [reply]
You can't use rigtools on 130. only 128 lower.
Check out my latest method i made a video on. It will work for you. 
https://youtu.be/td1VeHHhhb4?si=LIHrNRlIIVjZfMmS

**Reply by Noobster5** [reply]
i mean my other option is to downgrade but I don't want to do that.

**Comment by SlitheringLizard1** [comment]
blud if the blocks aren't there then the extension is disabled
you cannot have goguardian enabled and disabled
MAYBE you could hack into the admin's pc and remove the blocklist, but that would be illegal

**Comment by schoolexploitkid** [comment]
@Ryler4 use omada dns

**Reply by schoolexploitkid** [reply]
https://omada.cafe/#:~:text=Omada%20GitHub%20alternative-,OmadaDNS,-OmadaDNS%20is%20a
